moderation.php

```diff
@@ -2379,7 +2383,7 @@
 		verify_post_check($mybb->get_input('my_post_key'));
 
 		$mergepost = $mybb->get_input('mergepost', MyBB::INPUT_ARRAY);
-		if(count($mergepost) <= 1)
+		if(count($mergepost) <= 2)
```

Comment:
I think this is wrong, note the operator `<=` has a equal sign. I was unable to merge two posts together.